src/singlylinkedlist.c

```diff
@@ -365,3 +365,44 @@ int singlylinkedlist_foreach(SINGLYLINKEDLIST_HANDLE list, LIST_ACTION_FUNCTION
 
     return result;
 }
+
+LIST_ITEM_HANDLE singlylinkedlist_add_head(SINGLYLINKEDLIST_HANDLE list, const void* item)
+{
+    LIST_ITEM_HANDLE result;
+
+    /* Codes_SRS_LIST_02_001: [ If list is NULL then singlylinkedlist_add_head shall fail and return NULL. ]*/
+    if (list == NULL)
+    {
+        LogError("Invalid argument SINGLYLINKEDLIST_HANDLE list=%p, const void* item=%p", list, item);
```

Comment:
>, item [](start = 94, length = 6)

Putting the variable 'item' in the LogError will only add confusion since it doesn't matter if it's NULL or NOT.  Please Remove.

Reply:
Removed printing item

---
In reply to: [221642887](https://github.com/Azure/azure-c-shared-utility/pull/199#discussion_r221642887) [](ancestors = 221642887)